tests/unit/test_protocols.py

```diff
@@ -144,7 +142,7 @@ def _compliance_tests(test_type=None):
 def test_input_compliance(json_description, case, basename):
     service_description = copy.deepcopy(json_description)
     service_description['operations'] = {
-        case.get('name', 'OperationName'): case,
+        case.get('given', {}).get('name', 'OperationName'): case,
```

Comment:
Nit:

I'm guessing this is to handle the difference in formats between legacy and new ones?  It's fine as is, an alternative could be just standardizing the legacy ones so they have the same key to make this a little more readable.